README.md

```diff
@@ -35,6 +35,7 @@ ENV.string("APP_NAME", default: "local")
 ENV.symbol("BUSINESS_DOMAIN", default: :engineering, required: true)
 ENV.boolean("ENABLE_FEATURE_FOO", default: false)
 ENV.integer("MAX_THREAD_COUNT", default: 5)
+ENV.date("SCHEDULED_DATE", required: true, format: "%Y-%m-%d")
```

Comment:
Should the README document that `default` is available for `date`?

```suggestion
ENV.date("SCHEDULED_DATE", default: "2042-04-02", required: true, format: "%Y-%m-%d")
```

Reply:
That's not a valid default value, but it could. All of the helpers accept a `default` parameter, that's documented [here](https://github.com/nevinera/environment_helpers/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L42-L43); the examples in this code block can only exhibit `default` OR `required` in each case, since they don't make sense to use together.